dotnet/src/Discover.APIToolkit/Client.cs

```diff
@@ -29,7 +29,6 @@ private void Initialize()
 
             HttpClient = new HttpClient();
             HttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", Configuration.Token);
-            HttpClient.DefaultRequestHeaders.Add("ApiKey", Configuration.ApiKey);
```

Comment:
Why is this removed?

Reply:
The new ("V2") version of the auth/jwt token in Discover doesn't require the "key" component separately as it is baked into the token now.

Reply:
cool, thanks for confirming